### PR TITLE
Create Registration Lambda

### PR DESCRIPTION
**Pull request by sdrafahl**
https://zube.io/sdrafahl/terra/c/49

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,16 +5,26 @@ import sbtrelease.Version
 name := "hello"
 
 resolvers += Resolver.sonatypeRepo("public")
+resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
 scalaVersion := "2.12.8"
 releaseNextVersion := { ver => Version(ver).map(_.bumpMinor.string).getOrElse("Error") }
 assemblyJarName in assembly := "hello.jar"
 
+addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.0")
```
Comment (jd1388) [inline build.sbt:13]: What do the `%%` and `%` operators do?

File: project/.bloop/terraapi-build.json [hunk 2]
```diff
@@ -0,0 +1,1509 @@
+{
+    "version" : "1.1.0-M2",
+    "project" : {
+        "name" : "terraapi-build",
+        "directory" : "/home/shane/code/TerraApi/project",
+        "sources" : [
+            "/home/shane/code/TerraApi/project/src/main/scala-2.12",
```
Comment (jd1388) [inline project/.bloop/terraapi-build.json:7]: Shouldn't this be using relative filepaths? I'm not going to have `/home/shane/code/...`

**Review by jd1388 — COMMENTED** [review]
We should have a conversation about using Scala. I know that you want to use it because it's a new piece of tech that you enjoy using, but I can see problems in the future if Ben or I have to touch this codebase. Whiel I was able to point out some (potential) issues with the classpaths, I can confidently say I have no idea what's going on here. If I don't get what this is doing, there is no hope Ben will. That's not going to be a sustainable approach because we won't be able to effectively review this, and the solution won't be for Ben and I to "just learn Scala". There needs to be a team discussion about what tech to use. I don't think it needs to be JavaScript by any means, but something the three of us understand.

File: project/.bloop/terraapi-build.json (hunk 2 above)
Comment (sdrafahl) [reply project/.bloop/terraapi-build.json:7]: I need to remove this. I thought the bloop files were gitignored.

The Bloop files are auto generated.

File: build.sbt (hunk 1 above)
Comment (sdrafahl) [reply build.sbt:13]: Its just syntactic sugar for adding a version number to the end.

File: project/.bloop/terraapi-build.json (hunk 2 above)
Comment (sdrafahl) [reply project/.bloop/terraapi-build.json:7]: Its just junk

**Pull request merged**